### PR TITLE
Global header markup

### LINKED ISSUE #43: Greenpeace logo is fuzzy
screen shot 2017-11-27 at 3 26 28 pm

The Greenpeace logo is quite fuzzy on hi-DPI screens. This would be better as an SVG and also easier to style/control.

### LINKED ISSUE #60: Home page review
Hey all, I'll be posting reviews for each screen size of Home here. So far I got for you S and XL sizes. 
I screenshoted the published page and added comments in the pdf directly to make it as straight-forward as possible. Whoever will be taking this down (as assigned by Adrian) please see the attached files.
[Home_S.pdf](https://github.com/greenpeace/planet4-presentation-layer/files/1521705/Home_S.pdf)
[HomeXL.pdf](https://github.com/greenpeace/planet4-presentation-layer/files/1521706/HomeXL.pdf)

There's been a few design updates to it (mostly color and font-sizes). Latest master file and exported screens have been uploaded to [Drive](https://drive.google.com/drive/u/0/folders/0B-hrpY3C01WJaFhEYjlPeE52MkU).

Any questions, I'd suggest reaching out directly to me in Rocketchat, that's the quickest way. Cheers!

### PR DESCRIPTION
**Pull request by kadamwhite**
This is a big one: this PR re-envisions the markup for the top navigation header, removing a lot of duplication, standardizing behavior across breakpoints, and improving the accessibility and usability of the menu.

I'll add a more thorough write-up in the morning, but for now, the best way to get a feel for this is probably to pull it down and test it.

This PR supersedes #11 

Resolves #43 
Resolves #60

### CODE REVIEW

**Comment by kadamwhite** [comment]
Large breakpoint, country menu open:
image
Smaller breakpoint, country menu open:
image
Explore header:
image

Tablet size, search menu open:
image

Mobile:
image

Mobile menu:
&nbsp;

File: home.html [hunk 1]
```diff
@@ -10,717 +10,241 @@
...
-                        <button class="top-nav-search-btn" type="submit"><i class="icon-search"></i></button>
-                      </form>
-                    </li>
-                  </ul>
-                </div>
-              </nav>
-            </div>
-          </div>
+  <nav id="header" class="top-navigation">
+    <a class="site-logo" href="/">
```
Comment (mikeselander) [inline home.html]: Could we have this link to `home.html` or change this file to `index.html`? Index appears un-maintained and leads to confusion imo.

File: home.html [hunk 2]
```diff
@@ -10,717 +10,241 @@
...
+      <span class="navbar-toggle-label">Menu</span>
+    </button>
+    <ul id="navbar-dropdown" class="navbar-dropdown list-unstyled">
+      <li class="nav-item">
+        <button
+          class="country-dropdown-toggle"
+          data-toggle="open"
+          data-target="#country-list"
+          aria-expanded="false"
+          aria-label="Toggle worldwide site selection menu"
```
Comment (mikeselander) [inline home.html]: My understanding was that it was preferable to have a `title` attribute, but this could be off.

File: home.html [hunk 3]
```diff
@@ -10,717 +10,241 @@
...
+    </button>
+    <form id="navbar-search" class="form nav-item nav-search-wrap">
+      <input class="form-control" type="text" placeholder="Search" aria-label="Search">
+      <button class="top-nav-search-btn" type="submit">
+        <i class="fa fa-search"></i>
+        <span class="screen-reader-text">Search</span>
+      </button>
+    </form>
+  </nav>
+  <div class="hero-img-wrap">
```
Comment (mikeselander) [inline home.html]: Do we need this empty div?

File: home.html [hunk 4]
```diff
@@ -1122,8 +646,9 @@ <h2>Get action alerts in your inbox</h2>
 <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta/js/bootstrap.min.js" integrity="sha384-h0AbiXch4ZDo7tp9hKZ4TsHbi047NrKGLO3SEJAg45jXxnGIfYzk4Si90RDIqNm1" crossorigin="anonymous"></script>
 <script src="https://cdn.jsdelivr.net/npm/slick-carousel@1.8.1/slick/slick.min.js"></script>
 <!-- File for all the custom JS that we write -->
+<script type="text/javascript" src="src/js/header.js"></script>
```
Comment (mikeselander) [inline home.html]: Is this not compiled to custom.js like the rest of the JS?

**Comment by kadamwhite** [comment]
Issues:

- [ ] Lang dropdown occluded by cookie notice and carousel buttons
image
- [ ] Switch back to "Menu" on mobile/intermediate sizes; adjust Greenpeace logo positioning
- [ ] Make dropdown wider on medium screens
- [ ] Add 50% black overlay behind the dropdown menu
- [ ] Adjust hover and active states on the top

File: src/scss/common/_navbar.scss [hunk 5]
```diff
@@ -1,473 +1,520 @@
...
+  @include large-and-up {
+    margin-top: 11px;
+    max-width: 20%;
+    display: inline-block;
+  }
+
+  input::-webkit-input-placeholder {
+    color: $grey-40;
+  }
+  .top-nav-search-btn {
```
Comment (mikeselander) [inline src/scss/common/_navbar.scss]: Spacing inconsistencies here.

File: src/scss/common/_navbar.scss [hunk 6]
```diff
@@ -1,473 +1,520 @@
...
 }
 
-.country-icon {
-  width: 28px;
-  height: 28px;
-  float: left;
-  margin: 6px 10px 0 0;
+.navbar-search-toggle,
+.btn-navbar-toggle {
+  line-height: rem(54 - 22);
```
Comment (mikeselander) [inline src/scss/common/_navbar.scss]: Why not a simpler, unitless value?

The old styles had all of these hard-set line-heights and it made it way more difficult to handle when font-size increased or decreased so I'd prefer to use unitless on line-height wherever possible.

**Pull request merged**